### PR TITLE
Milestone3 review

### PR DESCRIPTION
**Pull request by Byabasaija**
In this milestone we,
- Implemented the add book functionality to the store
- Implemented the remove book functionality to the store
A user can therefore and multiple books to the store and also is able to remove/delete books from the store.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
+import { connect } from 'react-redux';
+import { bookRemoveAction } from '../actions';
 
 function Book(props) {
-  const { book } = props;
+  const { book, bookRemoveAction } = props;
+
+  const handleRemoveBook = (book) => {
+    bookRemoveAction(book);
+  };
```
Comment (trekab) [inline src/components/Book.js]: This method is expected to be implemented in the `BookList` component and passed down to the book as a prop. [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi team! Your work so far is commendable! :+1: However, there is just one requirement I'd suggest you look into before having your PR approved i.e.

- [ ] Check out the code comment below about your implementation of the `handleRemoveBook` method. ([screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png))

_Please make the necessary changes and submit another code review request._
__[Treasure Kabareebe](https://trekab.github.io)__

File: src/containers/BooksList.js [hunk 2]
```diff
@@ -1,29 +1,43 @@
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Book from '../components/Book';
+import { bookRemoveAction } from '../actions/index';
 
 const BookList = (props) => {
   const { books } = props;
```
Comment (Elbie-em) [inline src/containers/BooksList.js:8]: - [ ] Consider destructuring your props in a single line, this makes the code look more clean and professional.
       ` const { books , removeBook} = props   OR  const BookList = ({books, removeBook}) => {....}`

**Review by Elbie-em — APPROVED** [review]
## STATUS: APPROVED ✅ 

Hi @Byabasaija and @saimaAnis21, congratulations on your great effort in applying the changes requested. You can proceed to the next milestone.

### [OPTIONAL]
- [ ] Check my comments below to apply some optional changes

*All the best* 💪 

> Elbie Moonga (Part-time TSE)

> Github (@Elbie-em)
> Slack (Elbie Moonga)
> Twitter ([@ElbieEm](https://twitter.com/ElbieEm))
> LinkedIn([Elbie Moonga](https://www.linkedin.com/in/elbiemoonga/))

**Pull request merged**